src/dodal/common/enums.py

```diff
@@ -0,0 +1,31 @@
+from ophyd_async.core import StrictEnum
+
+
+class OnState(StrictEnum):
+    ON = "On"
+    OFF = "OFF"
```

Comment:
```suggestion
    OFF = "Off"
```

Looks like O**n** O**FF** wasn't used before, so just a slip here.

I'm checking with TC if an epics str pv is necessarily always case sensitive?

Reply:
Good spot